app/Http/Controllers/Cooperation/HomeController.php

```diff
@@ -24,7 +25,11 @@ public function index(Cooperation $cooperation)
         $building = HoomdossierSession::getBuilding(true);
 
         // If the quick scan is complete, we just redirect to my plan
-        if ($building->hasCompletedQuickScan()) {
+
+        // todo: figure out why there is no use of forInputSource master
```

Comment:
So this controller determines the redirect url, this now depends on the current input source and should be refactored later on